fleet-management-service/src/test/java/cz/fi/muni/pa165/service/JourneyServiceImplTest.java

```diff
@@ -125,11 +124,10 @@ public void testFinishJourney() {
         when(journeyDao.findById(any(Long.class))).thenReturn(journey1);
         journeyService.finishJourney(journey1.getId(),distance,endDate);
 
-        Journey expectedJourney = journey1;
-        expectedJourney.setDistance(distance);
-        expectedJourney.setReturnedAt(endDate);
+        journey1.setDistance(distance);
+        journey1.setReturnedAt(endDate);
 
-        verify(journeyDao).merge(expectedJourney);
+        verify(journeyDao).merge(journey1);
```

Comment:
Removal of `expectedJourney` was suggested by IDEA. I could have keep it... but whatever.